### PR TITLE
Update wikicontributing.rst

### PR DESCRIPTION
**Pull request by v-h-gn**
_Use the following template. Feel free to remove any unused sections._

## wikicontributing.rst
* Changes miscellaneous text and titles so page corresponds with Contribution guidelines.

### CODE REVIEW

File: source/general/wikicontributing.rst [hunk 1]
```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
 Setting up the environment
 ==========================
 
-To begin editing the wiki we first need to set our environment up. The prerequisites needed is just `python <https://www.python.org/downloads/>`_.
+To begin editing the Wiki we first need to up our environment. `Python`_ is the only dependency required.
 
-.. note:: When installing python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
+.. note:: When installing Python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: Change `sure too tick` to `sure to tick`.

File: source/general/wikicontributing.rst [hunk 2]
```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
 Setting up the environment
 ==========================
 
-To begin editing the wiki we first need to set our environment up. The prerequisites needed is just `python <https://www.python.org/downloads/>`_.
+To begin editing the Wiki we first need to up our environment. `Python`_ is the only dependency required.
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: > To begin editing the Wiki, you first need to set up our environment.

File: source/general/wikicontributing.rst [hunk 3]
```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
...
 
-.. note:: When installing python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
+.. note:: When installing Python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
 
-First thing you need to do is to fork the git repository and then clone it.
+
+1. Fork the GitHub repository and clone it.
 
 .. note:: Remember to replace *exampleusername* with your actual github username
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: Maybe make this all caps to make it more obvious?

File: source/general/wikicontributing.rst [hunk 4]
```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
...
 
-To check that sphinx is correctly install run:
+To check that Sphinx is correctly installed run:
 
 .. code:: bash
 
     sphinx-build -version
 
-If sphinx has been installed correctly it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
+If Sphinx has been installed correctly it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: > If Sphinx has been installed correctly, it should output sphinx-build and the version number.

File: source/general/wikicontributing.rst [hunk 5]
```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
...
 
-Once you have made some changes and you are ready to see your edits locally open terminal / command prompt and run
+Once you have made some changes and are ready to see your edits locally, open terminal / command prompt and run
 
 .. code:: bash
 
     .\make.bat html
 
-This should build a copy of the wiki into the build folder in the root directory of your repository.
-To view the wiki double click on ``build > html > index.html``
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst:51]: Maybe leave this code block path?

File: source/general/wikicontributing.rst [hunk 6]
```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
...
+This should build an HTML version of the wiki in the build folder in the root directory of your repository under the html folder.
 
-Submitting your changes
-=======================
+Submitting Changes
+==================
 
 Now that you are happy with your changes remember to commit them using git and make sure to put a detailed message indicating what you have done.
-If you are fixing/completing an issue please also reference the issue number in your commit message.
+If you are fixing/completing an issue please also reference the issue number/Trello card in your commit description.
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: Change `fixing/completing an issue` to `working on an issue,`
Change `the issue number/Trello card` to `the GitHub issue / Trello card`

File: source/general/wikicontributing.rst [hunk 7]
```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
...
 
-Remember to select compare across forks as shown in the screenshot below
+Remember to compare across forks as shown in the screenshot below
 
 .. image:: /_static/images/contributing_pr_1.png
 
-Once you are ready and content with your changes hit the *Create pull request* button
+Once you are ready and content with your changes hit the *Create pull request* button.
+
+Your changes will be reviewed and then merged by the Wiki Team. They may request changes and delay merging if they find any errors.
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst:67]: Mention filling out the PR template?

**Review by ChristopherWMM — CHANGES_REQUESTED** [review]

File: source/general/wikicontributing.rst (hunk 3 above)
Comment (jphuse) [reply source/general/wikicontributing.rst]: I feel that if the user is planning on contributing, they hopefully know to read everything on here, especially if they attempt to make a PR with changes.

File: source/general/wikicontributing.rst (hunk 5 above)
Comment (jphuse) [reply source/general/wikicontributing.rst:51]: Yeah, especially if someone doesn't know what they're doing, this is an easy way to learn.

File: source/general/wikicontributing.rst [hunk 8]
```diff
@@ -4,64 +4,66 @@
 Contributing to the Wiki
 ========================
 
-Setting up the environment
+Setting Up the Environment
 ==========================
 
-To begin editing the wiki we first need to set our environment up. The prerequisites needed is just `python <https://www.python.org/downloads/>`_.
+To begin editing the Wiki, you first need to your environment. `Python`_ is the only dependency required.
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst:10]: Change to `To begin editing the Wiki, you first need to set up its environment. Python is the only dependency required.`

File: source/general/wikicontributing.rst (hunk 3 above)
Comment (ChristopherWMM) [reply source/general/wikicontributing.rst]: Better safe than sorry, making it all caps makes it much more obvious.

File: source/general/wikicontributing.rst [hunk 9]
```diff
@@ -4,64 +4,66 @@
...
 
-To check that sphinx is correctly install run:
+To check that Sphinx is correctly installed, run:
 
 .. code:: bash
 
     sphinx-build -version
 
-If sphinx has been installed correctly it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
+If Sphinx has been installed correctly, it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: Missing period after `number`. Should be `version number. For example`

File: source/general/wikicontributing.rst (hunk 5 above)
Comment (ChristopherWMM) [reply source/general/wikicontributing.rst:51]: You should leave the code path.

File: source/general/wikicontributing.rst [hunk 10]
```diff
@@ -4,64 +4,66 @@
...
+This should build an HTML version of the wiki in the build folder in the root directory of your repository under the html folder.
 
-Submitting your changes
-=======================
+Submitting Changes
+==================
 
 Now that you are happy with your changes remember to commit them using git and make sure to put a detailed message indicating what you have done.
-If you are fixing/completing an issue please also reference the issue number in your commit message.
+If you are working on an issue, please include a link to the corresponding Trello Card/GitHub Issue in your commit description.
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: Change the `GitHub Issue / Trello Card` to keep consistency, you add spaces between all other slashes.

**Review by ChristopherWMM — CHANGES_REQUESTED** [review]
Almost finished up @Varhagna

File: source/general/wikicontributing.rst (hunk 8 above)
Comment (vtovmasian) [reply source/general/wikicontributing.rst:10]: I think your fits better in this scenario

Comment (ChristopherWMM) [reply source/general/wikicontributing.rst:10]: Change to `To begin editing the Wiki, you first need to set up its environment. Python is the only dependency required.`

File: source/general/wikicontributing.rst (hunk 3 above)
Comment (ChristopherWMM) [reply source/general/wikicontributing.rst]: Change to `YOUR_USERNAME`

File: source/general/wikicontributing.rst [hunk 11]
```diff
@@ -4,64 +4,66 @@
...
 
-First thing you need to do is to fork the git repository and then clone it.
+
+1. Fork the GitHub repository and clone it.
 
 .. note:: Remember to replace *exampleusername* with your actual github username
 
 .. code:: bash
 
     git clone https://github.com/exampleusername/ProjectKorraWiki.git
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: Change `exampleusername` to `YOUR_USERNAME`

File: source/general/wikicontributing.rst [hunk 12]
```diff
@@ -4,64 +4,66 @@
...
-Once you have made some changes and you are ready to see your edits locally open terminal / command prompt and run
+Once you have made some changes and are ready to see your edits locally, open terminal / command prompt and run
 
 .. code:: bash
 
     .\make.bat html
 
-This should build a copy of the wiki into the build folder in the root directory of your repository.
-To view the wiki double click on ``build > html > index.html``
+This should build an HTML version of the wiki in the build folder in the root directory of your repository under the html folder.
```
Comment (ChristopherWMM) [inline source/general/wikicontributing.rst]: Add (``build > html > index.html``) to the end of this line after the period.

**Review by ChristopherWMM — CHANGES_REQUESTED** [review]
Almost there buddy :sweat_smile:

**Review by ChristopherWMM — APPROVED** [review]

**Review by jphuse — APPROVED** [review]

**Pull request merged**